### PR TITLE
MTV-1709: Release Notes for MTV 2.7.5

### PR DESCRIPTION
**Pull request by anarnold97**
### JIRA

* [MTV-1709](https://issues.redhat.com/browse/MTV-1709)

### REVIEW

* [1.3.1. Resolved issues 2.7.5](https://anarnold97.github.io/MTA-Preview/MTV/mtv-rn-2-7-5.html#resolved-issues-2-7-5_release-notes)

* [1.4. Known issues](https://anarnold97.github.io/MTA-Preview/MTV/mtv-rn-2-7-5.html#known-issues-2-7_release-notes)

### CODE REVIEW

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 1]
```diff
@@ -7,6 +7,34 @@
...
+
+.Missing VM network-ID in the inventory
+
+In earlier releases of {project-short}, after creating a migration plan for VMs with a NSX-T network attached from vSphere, the VM network mapping was missing, and also adding network mapping could not list NSX-T networks as source networks. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1695[(MTV-1695)] and link:https://issues.redhat.com/browse/MTV-1140[(MTV-1140)]
+
+.VMs with `/dev/disk/by-path` can not migrate successfully
+
+In earlier releases of {project-short}, VMs could not be migrated successfully. This issue was caused by SUSE Linux using `/dev/disk/by-path` by default in their `/etc/fstab/` entry, which caused `virt-v2v` migrations to fail with errors as such: `virt-v2v: error: mount: mount_stub: /dev/disk/by-path/pci-0000:03:00.0-scsi-0:0:0:0-part3: No such file or directory`. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1660[(MTV-1660)]
+
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:24]:
```suggestion

```

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 2]
```diff
@@ -7,6 +7,30 @@
...
+.Failure to create Windows 2019 VM during Cold Migration
+
+In earlier releases of {project-short}, cold migrating Windows Server 2019 VM from {rhv-full} ({rhv-short}) to a remote cluster returned a `firmware.bootloader setting` error of `admission webhook "virtualmachine-validator.kubevirt.io" denied the request` during the *VirtualMachineCreation* phase. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1613[(MTV-1613)]
+
+.Migration of a VM with Secure Boot enabled VM migration results in a VM with Secure Boot disabled
+
+In earlier releases of {project-short}, when migrating a virtual machine (VM) with *Secure Boot* enabled, the VM had *Secure Boot* as `disabled` after being migrated. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1632[(MTV-1632)]
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:27]:
```suggestion
```

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 3]
```diff
@@ -7,6 +7,26 @@
...
+[id="resolved-issues-2-7-5_{context}"]
+== Resolved issues 2.7.5
+
+.XFS filesystem corruption after warm migration of VM from VMware
+
+In earlier releases of {project-short}, virtual machines (VM) were presenting XFS filesystem corruption after the warm migration from VMware to {virt} using {project-short}. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1679[(MTV-1679)]
+
+.Missing VM network-ID in the inventory
+
+In earlier releases of {project-short}, after creating a migration plan for VMs with a NSX-T network attached from vSphere, the VM network mapping was missing, and also adding network mapping could not list NSX-T networks as source networks. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1695[(MTV-1695)] and link:https://issues.redhat.com/browse/MTV-1140[(MTV-1140)]
```
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:19]: Am I ok to combine these two Jira tickets into one entry? 

I can see [MTV-1140](https://issues.redhat.com/browse/MTV-1140) is linked to [MTV-1695](https://issues.redhat.com/browse/MTV-1695)

image

File: documentation/modules/known-issues-2-7.adoc [hunk 4]
```diff
@@ -16,9 +16,10 @@ include::snip_secure_boot_issue.adoc[]
 
 include::snip_measured_boot_windows_vm.adoc[]
 
+//do you want me to label this as resolved in 2.7.5 or delete as now included in resolved issues
 .Migration of a VM with Secure Boot enabled VM migration results in a VM with Secure Boot disabled
 
-When migrating a virtual machine (VM) with *Secure Boot* enabled, the VM has *Secure Boot* as `disabled` after being migrated. link:https://issues.redhat.com/browse/MTV-1632[(MTV-1632)]
+When migrating a virtual machine (VM) with *Secure Boot* enabled, the VM has *Secure Boot* as `disabled` after being migrated. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1632[(MTV-1632)]
```
Comment (fabiand) [inline documentation/modules/known-issues-2-7.adoc:22]: Tihs is actually a known issue. https://issues.redhat.com/browse/MTV-1632 is the tracker for the bug.

tldr: SecureBoot appears to be disabled, due to the bug above, but in reality it is enabled.

File: documentation/modules/rn-27-resolved-issues.adoc [hunk 5]
```diff
@@ -7,6 +7,26 @@
 
 {project-first} 2.7 has the following resolved issues:
 
+[id="resolved-issues-2-7-5_{context}"]
+== Resolved issues 2.7.5
+
+.XFS filesystem corruption after warm migration of VM from VMware
+
+In earlier releases of {project-short}, virtual machines (VM) were presenting XFS filesystem corruption after the warm migration from VMware to {virt} using {project-short}. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1679[(MTV-1679)]
```
Comment (fabiand) [inline documentation/modules/rn-27-resolved-issues.adoc:15]: were presenting XFS - and other data - corruption

**Review by fabiand — COMMENTED** [review]
two nits, goo otherwise

File: documentation/modules/known-issues-2-7.adoc [hunk 6]
```diff
@@ -16,6 +16,7 @@ include::snip_secure_boot_issue.adoc[]
 
 include::snip_measured_boot_windows_vm.adoc[]
 
+//do you want me to label this as resolved in 2.7.5 or delete as now included in resolved issues
```
Comment (anarnold97) [inline documentation/modules/known-issues-2-7.adoc:19]:
```suggestion
```

File: documentation/modules/rn-27-resolved-issues.adoc (hunk 5 above)
Comment (anarnold97) [inline documentation/modules/rn-27-resolved-issues.adoc:15]:
```suggestion
In earlier releases of {project-short}, virtual machines (VM) were presenting XFS filesystem and other data corruption after the warm migration from VMware to {virt} using {project-short}. This issue has been resolved in {project-short} 2.7.5. link:https://issues.redhat.com/browse/MTV-1679[(MTV-1679)]
```

**Review by fabiand — APPROVED** [review]
